client/ciaos/test.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 import requests
 from unittest.mock import patch, Mock
-from client import Ciaos, Config
+from ciaos.client import Ciaos, Config
```

Comment:
from client was easier right? Can you update the how the users should import according to your changes?

Reply:
hey so this changes do not affect the way the client is being used by the customer
```python
from ciaos import Ciaos,Config
```
will remain  unchanged just like the current guide  says, these changes are only for encountering the issues that occur when building and publishing python library

Reply:
Guess you have to raise this on python-sdk.

Reply:
@noobed-max